### DISCUSSION TITLE
SQL file with schema and data [Apr 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5565] [Discord Threads]
> 🙋‍♂️ **dev_dipper**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### easiest way is to dump it into a database schema and connect that to nocodb. we do allow you to connect to your own DB 🙂  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/dashboard#connecting-to-external-database  
#  
> 🙋‍♂️ **dev_dipper**  
#### Hey <@310157751395287040> good to hear from you!  
#  
> 🙋‍♂️ **dev_dipper**  
#### yes thank you, I already learned how to connect an external db to nocodb, it's such a great option!  
#  
> 🙋‍♂️ **dev_dipper**  
#### regarding the way to dump it into a database schema, could you point me in a direction or few actionable bullet points?  
#  
> 💁 **o1lab**  
#### you can look it up in google 🙂  
#  
> 🙋‍♂️ **dev_dipper**  
#### yes that's right thanks a lot <@310157751395287040>  😆  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1088956343957852330)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
